homeassistant/components/tuya/sensor.py

```diff
@@ -572,9 +572,25 @@ class TuyaSensorEntityDescription(SensorEntityDescription):
     # Irrigator
     # https://developer.tuya.com/en/docs/iot/categoryggq?id=Kaiuz1qib7z0k
     "ggq": BATTERY_SENSORS,
-    # Water Detector
+    # Water Detector - Rain Sensor RB-SRAIN01
     # https://developer.tuya.com/en/docs/iot/categorysj?id=Kaiuz3iub2sli
-    "sj": BATTERY_SENSORS,
+    "sj": (
+        TuyaSensorEntityDescription(
+            key=DPCode.RAIN_SENSOR_VOL,
+            translation_key="rain_sensor_vol",
+            device_class=SensorDeviceClass.VOLTAGE,
+            state_class=SensorStateClass.MEASUREMENT,
+            native_unit_of_measurement=UnitOfElectricPotential.MILLIVOLT,
```

Comment:
Is there any reason we override the units here instead of getting the units from the device/API?